### ISSUE TITLE
**aMUSEd**  - a new masked image model based on MUSE 🔥

### ISSUE THREAD

**Issue by sayakpaul** [huggingface/diffusers#6458]
Today, we're excited to bring you aMUSEd - a lightweight masked-image-model (MIM) intended for fast generation 🚀

We're releasing aMUSEd as a research release because its quality is NOT SoTA.

Training code, paper, usage, fine-tuning code 👇
https://huggingface.co/blog/amused

![image](https://github.com/huggingface/diffusers/assets/22957388/c5fc7361-46f6-4dd3-981b-ed4957d8d4e7)
